ark/utils/data_utils_test.py

```diff
@@ -345,10 +238,10 @@ def test_generate_deepcell_input():
         fov1 = io.imread(fov1path)
         fov2 = io.imread(fov2path)
 
-        assert np.all(fov1[:, :, 0] == 3)
-        assert np.all(fov1[:, :, 1] == 4)
-        assert np.all(fov2[:, :, 0] == 2)
-        assert np.all(fov2[:, :, 1] == 2)
+        assert np.all(fov1[:, :, 0] == data_xr.loc['fov1', :, :, nucs].sum(dim='channels').values)
```

Comment:
Probably split this into two lines: first line sums the channels, second line compares summed values to value in the xarray. and use np.array_equal